argo/providers/base/provider.go

```diff
@@ -15,8 +15,29 @@ type PipelineDefinition struct {
 	BeamArgs        map[string][]string `yaml:"beamArgs"`
 }
 
+type ExperimentDefinition struct {
+	Name        string `yaml:"name"`
+	Version     string `yaml:"version"`
+	Description string `yaml:"description"`
+}
+
+type RunConfigurationDefinition struct {
+	Name            string `yaml:"name"`
+	Version         string `yaml:"version"`
+	PipelineName    string `yaml:"pipelineName"`
+	PipelineVersion string `yaml:"pipelineVersion"`
+	ExperimentName  string `yaml:"experimentName"`
+	Schedule        string `yaml:"schedule"`
```

Comment:
Shall we add the runtime parameters here even if they are not used by the KFP implementation (#175)?

Reply:
let's do it when we address that card